core/src/main/scala/kanaloa/reactive/dispatcher/queue/QueueProcessor.scala

```diff
@@ -52,22 +57,23 @@ class QueueProcessor(
 
     case RouteeFailed(ex) ⇒
       inflightCreations -= 1
-      log.error(ex, "Failed to retrieve Routee")
+      log.warning("Failed to retrieve Routee: " + ex.getMessage)
```

Comment:
Fair enough, I do think however, we need to put an error or warning (something) in place for when our Worker count breaches the minimum.